samples/core/xgboost_training_cm/README.md

```diff
@@ -10,10 +10,13 @@ Finally, a delete cluster operation runs to destroy the cluster it creates
 in the beginning. The delete cluster operation is used as an exit handler, meaning it will run regardless of whether the pipeline fails
 or not.
 
-## Requirements
-
-Preprocessing uses Google Cloud DataProc. Therefore, you must enable the [DataProc API](https://cloud.google.com/endpoints/docs/openapi/enable-api) for the given GCP project.
+## Permission
```

Comment:
Just IMO, `Requirements` sounds clearer and it's better to be in sync.

If we decide to change, also sync changes in https://github.com/kubeflow/pipelines/blob/8998087f4baefa1fb1780475c261874dee790b68/backend/src/apiserver/config/sample_config.json#L4 please.

Reply:
That's right. However if we change sample_config.json, we might need another backend image release to bake that in MKP deployment. WDYT?

Reply:
* prefer merge Requirements and Permission as uniformed with TFX, eight merge into one section, or structured. 

Preferred structure:
# Requirements
## Enable API
## Permission

Reply:
Another temporary solution if we want to change backend in a later release:

## Requirements
...

## Permission
Please read [#requirements section](#requirements)

Reply:
I'm okay with either, you can decide.

Reply:
Let me put a working item on the triaging issue, and temporarily use the current section title. Either way we need some follow-up work. Thanks @Bobgy !